README.md

```diff
@@ -1,56 +1,160 @@
+
+1. Basically, taking a number as input for the format method instead of a date object significantly simplifies the scope of this proposal while it still fully addresses the main objective which is to provide i18n building blocks to address this problem realm.
+2. Taking a date object means we should implement the comparison logic (relative time is about date distance between target and source dates). The source date is usually *now*, but not always. We would have to address modifying that. See [#4](https://github.com/tc39/proposal-intl-relative-time/issues/4).
+3. Taking a date object also means we should allow for different calendar calculations, which implies *Date* should support it. See [#6](https://github.com/tc39/proposal-intl-relative-time/issues/6) and [#13](https://github.com/tc39/proposal-intl-relative-time/issues/13).
+4. Taking a date object suggests we should be able to implement a *bestFit* algorithm, which has its own API challenges with respect to standardizing an approach that works for all cases. See [#7](https://github.com/tc39/proposal-intl-relative-time/issues/7), [#14](https://github.com/tc39/proposal-intl-relative-time/issues/14), and [#15](https://github.com/tc39/proposal-intl-relative-time/issues/15). 
+
+#### Difference between this and UnitFormat
+
+The fundamental difference between RelativeTimeFormat and UnitFormat is that RelativeTimeFormat displays a relative unit (e.g., `5 days ago` or `in 5 days`) while the UnitFormat displays an absolut unit (e.g., `-5 meters` or `5 meters`). Note that RelativeTimeFormat uses different internationalized messages based on the value sign direction, while UnitFormat uses the same internationalized message for all values.
+
+##### Count downs, e.g., 15 days, 0 hours, 27 minutes, and 52 seconds
+
+A count down for example is a mix of UnitFormat and ListFormat and is not a RelativeTimeFormat.
+
+### API
+
+#### Intl.RelativeTimeFormat([locales[, options]])
+
+The *Intl.RelativeTimeFormat* object is a constructor for objects that enable language sensitive reltive time formatting.
+
+#### locales
+
+Optional. A string with a BCP 47 language tag, or an array of such strings. For the general form and interpretation of the `locales` argument, see the [Intl page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation).
+
+#### options
+
+Optional. An object with some or all of the following properties:
+
+##### options.style
+
+The length of the internationalized message. Possible values are: *"long"* (default, e.g., `in 1 month`); *"short"* (e.g., `in 1 mo.`), or "narrow" (e.g., `in 1 mo.`). The narrow style could be similar to the short style for some locales.
+
+#### Example
+
+```js
+// Create a relative time formatter in your locale.
+let rtf = new Intl.RelativeTimeFormat("en", {style: "short"});
+```
+
+### Intl.RelativeTimeFormat.prototype.format(value, unit)
```

Comment:
Absolutely! Thanks. Let's include this in a next iteration/PR. Keeping track of it in #37